1.chunkedgraph.md

```diff
@@ -83,7 +85,7 @@ Pods should now be in `Running` status, provided there were no issues. Run the f
 ```shell
 $ kubectl exec -ti deploy/master -- bash
 // now you're in the container
-> ingest graph <unique_test_bigtable_name> datasets/test.yml --test
+> ingest graph <unique_test_bigtable_name> datasets/test.yml --test // avoid dashes in <unique_test_bigtable_name>, neuroglancer can't handle them
```

Comment:
I think it might be cave code that has issues with dashes. It is better to just say use underscores instead.